public/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Amazon Item Details Proxy Server</title>
+    <link rel="stylesheet" href="http://localhost:3001/style.css">
+    <link rel="stylesheet" href="http://localhost:3002/style.css">
+    <link rel="stylesheet" href="http://localhost:3003/style.css" />
+    <link rel="stylesheet" href="http://localhost:3004/style.css" />
```

Comment:
For any of the services that are  using css-modules or styled components, you shouldn't need to load the css files explicitly.